tests/integration/devfile/cmd_devfile_catalog_test.go

```diff
@@ -46,6 +47,17 @@ var _ = Describe("odo devfile catalog command tests", func() {
 			}
 			helper.MatchAllInOutput(output, wantOutput)
 		})
+		It("should list components successfully even with an invalid kubeconfig path or path points to existing directory", func() {
+			projectKC := os.Getenv("KUBECONFIG")
```

Comment:
A name like `originalKC` would be clearer

Reply:
Ok i guess that makes sense